rules/linux/macos_local_account.yml

```diff
@@ -0,0 +1,46 @@
+title: Local System Accounts Discovery
+id: ddf36b67-e872-4507-ab2e-46bda21b842c
+status: experimental
+description: Detects enumeration of local systeam accounts on MacOS
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/08
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1087.001/T1087.001.md
+logsource:
+  category: process_creation
+  product: macos
+detection:
+  selection_1:
+    ProcessName|endswith:
+      - '*/dscl'
+    CommandLine|contains:
+      - '. list /users'
```

Comment:
I would rather split it to the following:

```
    CommandLine|contains|all:
      - 'list'
      - '/users'
```

That would make it more stable and hard to bypass. In the current state you could bypass it just adding an extra space (:

Reply:
100% agree!